tests/Support/SupportArrTest.php

```diff
@@ -210,6 +213,11 @@ public function testGet()
 
     public function testHas()
     {
+        $array = ['products.desk' => ['price' => 100]];
+        $this->assertTrue(Arr::has($array, 'products.desk'));
+        $this->assertTrue(Arr::has($array, ['products.desk']));
+        $this->assertFalse(Arr::has($array, ['products.desk', 'missing']));
```

Comment:
In 5.2 the second argument to `has` should be a string or an integer, not an array. Arrays were supported for 5.3 only.

@taylorotwell that's why the build fails.